[2024-01-17T23:25:10.219+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T23:23:51.547580+00:00 [queued]>
[2024-01-17T23:25:10.232+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T23:23:51.547580+00:00 [queued]>
[2024-01-17T23:25:10.232+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-17T23:25:10.257+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): coinMarket> on 2024-01-17 23:23:51.547580+00:00
[2024-01-17T23:25:10.264+0000] {standard_task_runner.py:57} INFO - Started process 10576 to run task
[2024-01-17T23:25:10.269+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Stream_Data_5', 'coinMarket', 'scheduled__2024-01-17T23:23:51.547580+00:00', '--job-id', '1403', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream_5.py', '--cfg-path', '/tmp/tmp4ldvm9ct']
[2024-01-17T23:25:10.271+0000] {standard_task_runner.py:85} INFO - Job 1403: Subtask coinMarket
[2024-01-17T23:25:10.363+0000] {task_command.py:415} INFO - Running <TaskInstance: Stream_Data_5.coinMarket scheduled__2024-01-17T23:23:51.547580+00:00 [running]> on host 614a419ad930
[2024-01-17T23:25:10.491+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='Stream_Data_5' AIRFLOW_CTX_TASK_ID='coinMarket' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T23:23:51.547580+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-17T23:23:51.547580+00:00'
[2024-01-17T23:25:20.640+0000] {logging_mixin.py:151} INFO - HTTPSConnectionPool(host='pro-api.coinmarketcap.com', port=443): Max retries exceeded with url: /v1/cryptocurrency/listings/latest?start=1&convert=USD (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f6d3e3a37c0>: Failed to resolve 'pro-api.coinmarketcap.com' ([Errno -3] Temporary failure in name resolution)"))
[2024-01-17T23:25:20.640+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-17T23:25:20.651+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Stream_Data_5, task_id=coinMarket, execution_date=20240117T232351, start_date=20240117T232510, end_date=20240117T232520
[2024-01-17T23:25:20.699+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-17T23:25:20.719+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
